quinn-proto/src/lib.rs

```diff
@@ -115,6 +117,25 @@ pub mod fuzzing {
             })
         }
     }
+
+    #[derive(Debug)]
+    pub struct PacketParams {
+        pub local_cid_len: usize,
+        pub buf: BytesMut
+    }
+
+    impl Arbitrary for PacketParams {
+        fn arbitrary(u: &mut Unstructured<'_>) -> Result<Self> {
+            let local_cid_len: usize = usize::arbitrary(u)?;
+            let bytes: Vec<u8> = Vec::arbitrary(u)?;
+            let mut buf = BytesMut::with_capacity(bytes.len());
```

Comment:
I think you can just use `BytesMut::new()` here, since it'll do the equivalent allocation on `put_slice()` anyway.